### PR TITLE
Add coverage report and more badges

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve? 

1. add unit test coverage, use https://codecov.io/
2. add more badges

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -84,12 +85,10 @@ tidy:
 check: fmt lint check-static tidy
 
 coverage:
-	GO111MODULE=off go get github.com/wadey/gocovmerge
-	gocovmerge "$(TEST_DIR)"/cov.* | grep -vE ".*.pb.go" > "$(TEST_DIR)/tmp_unit_cov.out"
-	grep -v "$(CDC_PKG)/cdc/kv/testing.go" "$(TEST_DIR)/tmp_unit_cov.out" >"$(TEST_DIR)/unit_cov.out"
+	GO111MODULE=off go get github.com/zhouqiang-cl/gocovmerge
```
Comment (july2993) [inline Makefile]: what's the different with the origin version

Comment (amyangfei) [reply Makefile]: The new version fixes a bug when merging multiple coverage files (https://github.com/zhouqiang-cl/gocovmerge/commit/5256314471af44fe8bccbf0a1db1dd1f85637c1a)

**Pull request merged**